### PR TITLE
[16.0][MIG] purchase_return: Migration to 16.0.

### PR DESCRIPTION
**Pull request by PauBForgeFlow**

### CODE REVIEW

**Review by mariadforgeflow — APPROVED** [review]
LGTM! Code and functional review.

File: purchase_return/models/purchase_return_order_line.py [hunk 1]
```diff
@@ -126,21 +118,21 @@ class PurchaseReturnOrderLine(models.Model):
...
+        [("product", "Product"), ("line_section", "Section"), ("line_note", "Note")],
         default=False,
         help="Technical field for UX purpose.",
     )
 
     _sql_constraints = [
         (
             "accountable_required_fields",
-            "CHECK(display_type IS NOT NULL OR (product_id IS NOT NULL "
+            "CHECK(display_type IS NOT product OR (product_id IS NOT NULL "
```
Comment (BernatPForgeFlow) [inline purchase_return/models/purchase_return_order_line.py]: You must enclose the product value in single quotes. It should be like this: 'product'

**Review by BernatPForgeFlow — CHANGES_REQUESTED** [review]
Also, 'Send Return PO by email' button seems to not work properly. To open this wizard that composes the email, I would perhaps recommend you to look for a more up-to-date example and, if necessary, change the implementation.

Remember that to refer a record:
_Old_: ir_model_data.get_object_reference(
                    "purchase_return", "email_template_edi_purchase_return"
                )[1]
_New_: self.env.ref("purchase_return.email_template_edi_purchase_return")

File: purchase_return/README.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+===============
```
Comment (BernatPForgeFlow) [inline purchase_return/README.rst:1]: The README file does not need to be added, it is automatically generated from the files in the 'readme' folder.

**Review by HviorForgeFlow — APPROVED** [review]

**Pull request merged**